### ISSUE TITLE
The emoji in TextInput is gray

### ISSUE THREAD

**Issue by Dovelong0713** [slint-ui/slint#5171]
How TextInput supports emoji colors

**Comment by tronical** [comment]
Hi! Which platform are you on?

**Comment by Dovelong0713** [comment]
I am Windows 10,
Your question makes me feel that TextInput already supports emoji colors, but it's just my environment causing the exception, isn't it?

**Comment by tronical** [comment]
Yes that's right. I think I recall seeing this before on Linux but there the issue was that the font didn't provide any color glyphs.

Perhaps there's a bug in the way we select fonts and we end up picking the wrong one.

I wonder if this is reproducible with cargo-ui.

As a workaround, could you try enabling the renderer-skia feature and set the SLINT_BACKEND environment to variable to winit-skia?

**Comment by tronical** [comment]
I can reproduce this with cargo-ui on Windows with the FemtoVG renderer (works with Skia, but that uses a different font fallback mechanism). Somehow we end up picking a font that has the emojis but they're grey-scaled.

I do think that's a problem that will go away once we switch over to Fontique. Maybe as part of the parley work, perhaps we can do that earlier as well.

**Comment by tronical** [comment]
This should be resolved with fontique and improved glyph selection.
